charts/datadog/values.yaml

```diff
@@ -384,6 +384,12 @@ datadog:
     #
     enabled: false
 
+    ## @param scrubbing - boolean - required
+    ## Enable the scrubbing of containers for sensitive information.
+    ## The container scrubbing is taking significant resources during data collection.
```

Comment:
let's be more specific, this is for the kubernetes resources YAML, not just any containers. And after "The container scrubbing is taking significant resources during data collection." we can say "if you notice that the cluster-agent uses too much CPU in larger clusters turning this option off will improve the situation.

Reply:
Makes sense for the second part. I will update this 👍 
But we only scrub `container` and `initContainers`. I could make the text to something like: `enable the scrubbing of containers in the kubernetes resource YAML...`